src/Servers/IIS/IntegrationTesting.IIS/src/IISDeployer.cs

```diff
@@ -45,6 +46,11 @@ public IISDeployer(IISDeploymentParameters deploymentParameters, ILoggerFactory
 
         public override void Dispose()
         {
```

Comment:
What is this for?

Reply:
Making sure we don't dispose twice. I was seeing this when trying some stress tests locally and I think it should be fixed (though it isn't relevant to this pr).